app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -173,16 +96,28 @@ class RoutePresenter(
         }
     }
 
-    private fun showUnlockFallback(action: RouteAction.UnlockFallbackDialog) {
-        val manager = activity.getSystemService(Context.KEYGUARD_SERVICE) as KeyguardManager
-        val intent = manager.createConfirmDeviceCredentialIntent(
-            activity.getString(R.string.unlock_fallback_title),
-            activity.getString(R.string.confirm_pattern)
-        )
-        currentFragment.startActivityForResult(intent, action.requestCode)
+    fun showAutofillDialogFragment(dialogFragment: DialogFragment, destination: RouteAction.DialogFragment) {
```

Comment:
We need both versions of `showDialogFragment` (one specifically for autofill). Otherwise autofill will fail on the line 
```dialogFragment.setTargetFragment(currentFragment, 0)```